### DISCUSSION TITLE
library(tidyr)等包时出现错误错误: package or namespace load failed for ‘tidyr’ in loadNamespace

### DISCUSSION THREAD

**Discussion by JYL2303** [ShixiangWang/self-study#48] [Q&A] [answered]
> library(tidyr)
错误: package or namespace load failed for ‘tidyr’ in loadNamespace(j <- i[[1L]], c(lib.loc, .libPaths()), versionCheck = vI[[j]]):
 载入了名字空间‘vctrs’ 0.2.2，但需要的是>= 0.2.4

好几个包都出现了相同的问题，如broom, dplyr, mice,tidyverse, gtsummary.
问题的来源于拜读了您关于gtsummary （https://mp.weixin.qq.com/s/NYxZAkVJiT9fyz684KU6Kw）
的文章，安装gtsummary的时候出现了问题，后来重新安装成功后，library就出现了相关的问题，接着发现其他的包也出现了类似的错误，其他包昨天还用过，所以应该是今天安装gtsummary的问题。
尝试更新R以及重新装了Rstudio，但是问题依然存在，特来请教。感谢解答。

**Comment by ShixiangWang** [comment, marked as answer]
先清除环境空间：

```
rm(list=ls()
```

重启 RStudio，保存空间。

安装 `vctrs` 包。

然后再看看问题是否还存在

**Comment by JYL2303** [comment]
倒腾来倒腾去，已经好了，感谢~

**Comment by ShixiangWang** [comment]
好👌
